pySecDec/code_writer/make_package.py

```diff
@@ -559,7 +561,7 @@ def _make_FORM_shifted_orders(positive_powers):
     r'''
     Write FORM code that defines the preprocessor
     variables
-    `shiftedRegulator`regulatorIndex'PowerOrder`shiftedOrderIndex''.
```

Comment:
This changes the variable name (so the comment is no longer correct).
Please remove the change to this line.

Reply:
ok, undone. But this line is not in correct rst syntax. You may like to fix it later.